high_card.rb

```diff
@@ -1,5 +1,12 @@
 # Use these two arrays to generate a deck of cards.
-ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
-suits = [ "hearts", "spades", "clubs", "diamonds" ]
 
-players = []
+
+ranks = %i["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
+suits = %i[ "hearts", "spades", "clubs", "diamonds" ]
+
+
+deck = ranks.product(suits).shuffle
```

Comment:
Nice use of `.product`. Haven't actually tried that one out myself, but I will now!